python-base/ubuntu20.04-python3.9/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM ubuntu:20.04
+FROM ubuntu:ubuntu20.04
```

Comment:
What does this do?

Reply:
I created a new ubuntu image with an updated `OpenSSL` and I am now extending the base images from the new ubuntu image.

Reply:
No it's not.